packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -114,10 +115,18 @@ class BondWithdrawalWatcher extends BaseWatcher {
     }
 
     let availableCredit = await destBridge.getAvailableCredit()
-    const includePendingAmount = destinationChain === Chain.Ethereum && [Chain.Optimism, Chain.Arbitrum].includes(sourceChain)
+    const includePendingAmount = destinationChain === Chain.Ethereum && bondableChains.includes(sourceChain)
     if (includePendingAmount) {
-      const pendingAmount = await sourceL2Bridge.getPendingAmountForChainId(destinationChainId)
-      availableCredit = availableCredit.sub(pendingAmount).sub((amount).mul(2))
+      let pendingAmounts = BigNumber.from(0)
+      for (const chain of bondableChains) {
+        const bridge = this.getSiblingWatcherByChainSlug(chain)
```

Comment:
I believe this needs a `.bridge`. Probably a `.bridge as L2Bridge` since the pending amount check will only be on the L2 Bridge.

Reply:
good catch, fixed